wandb/internal/update.py

```diff
@@ -63,3 +74,12 @@ def check_available(current_version):
             " --pre" if pip_prerelease else "",
         )
     )
+
+    yank_message = None
+    if yanked:
+        yank_message = "WARNING: your current version has been YANKED"
+    if deleted:
+        yank_message = "WARNING: your current version has been DELETED"
```

Comment:
nit -- it's bad practice to rely on the order of the `if` statements to ensure that the `deleted` case overwrites the `yanked` case if somehow they're both `True`. I know that it's currently impossible for that to happen, but I think the (subjectively) best way to do this is:

```
if deleted: #highest priority first
elif yanked:
```

I think this maps well to most people's mental model of understanding: Check for the highest priority condition, else go down in descending priority. If a higher priority condition is met, they can skip reading the rest of the branches.

Reply:
got it, makes sense